roles/openshift_prometheus/templates/prometheus.yml.j2

```diff
@@ -237,6 +237,25 @@ scrape_configs:
     action: keep
     regex: apiserver;https
 
+# Scrape config for the router
+- job_name: 'openshift-router'
+  scheme: https
```

Comment:
In my 3.9 setup this worked only when I changed to `http`. Is this a router config setting?

Reply:
IIUC https://github.com/openshift/origin/pull/18254/commits/a63df7bc2a403204ddbc1818d05df32b8489e711 is the change that switched the metrics endpoint to HTTPS. And it isn't part of in 3.9.